### ISSUE TITLE
Rendering Video element

### ISSUE THREAD

**Issue by carlonluca** [nezticle/qml_movie_renderer#2]
I tried to render a QML scene including a Video element, but it seems the video starts only after the rendering process is completed. The result is just a black frame. Is this a known limitation?

**Comment by moritz-h** [comment]
I played a bit with this tool. I think the limitation is the playback speed of the Video, which is still in real time. When you play with the parameters to increase rendering time to something near real time (I used 200 FPS with 720p) you can actually see the video content in the output.

**Comment by carlonluca** [comment]
But assuming you render in realtime, do you get the video in the output? It seemed I couldn't get it.

**Comment by moritz-h** [comment]
Yes I get Video output in the images I write. I have a very simple test qml. Just a black Rectangle and a Video element inside, both with width and height set. I verified, that the video generally could be played by running it with qmlscene.
I run with settings: duration 5s and 200 FPS. Output as png. Then I get around 100 black frames at the start (hard to say exactly, testing video starts with fade in from black), but then there is the rendered video visible in the png output.
I'm testing on Ubuntu 16.04 with the Qt 5.9.1 from the Online installer.

(Perhaps for trying this Qt example is helpful: https://github.com/qt/qtdeclarative/tree/5.9/examples/quick/rendercontrol The examples also renders qml to virtual framebuffer, but for displaying in realtime, so the animation control part is not needed. Adding a Video to the qml works perfectly there.)
